### PR TITLE
Hr watch leave 

### PR DESCRIPTION
**Pull request by shivamchouhan123**
if hr want to see particular attendences and leaves other wise by default show all employee data

### CODE REVIEW

File: app/controllers/api/v1/attendences_controller.rb [hunk 1]
```diff
@@ -9,10 +9,22 @@ class AttendencesController < ApplicationController
       before_action :authenticate_employee!
 
       def index
+        if current_employee.is_hr?   
+          attendence_data = if params[:employee_id].present? 
+                              Attendence.where(employee_id: params[:employee_id]).order(id: :desc)          
+                            else
+                              Attendence.all.order(id: :desc)
```
Comment (ror-mahesh) [inline app/controllers/api/v1/attendences_controller.rb:16]: refactor the code..

**Review by ror-mahesh — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/attendences_controller.rb (hunk 1 above)
Comment (shivamchouhan123) [reply app/controllers/api/v1/attendences_controller.rb]: resolve

File: app/controllers/api/v1/leafs_controller.rb [hunk 2]
```diff
@@ -7,16 +7,24 @@ class LeafsController < ApplicationController
...
+                      else
+                        Leaf.all.order(id: :desc)  
+                      end
+          render json: {
+            data: serializer_data(leaf_data, leaf_serializer),
+            message: ['show leaves '], status: 200, type: 'Success'
+          }
+        else
+          leaf_data = current_employee.leafs.order('id desc')
+            render json: {
```
[the remark this answers is not in the document]
Comment (shivamchouhan123) [reply app/controllers/api/v1/leafs_controller.rb:18]: resolve

**Review by ror-mahesh — APPROVED** [review]
LGTM

**Pull request merged**